2023.04-4week/wintaek/120823. 직각삼각형 출력하기/직각삼각형 출력하기.js

```diff
@@ -0,0 +1,14 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let input = [];
+
+rl.on("line", function (line) {
+  input = line.split(" ");
+}).on("close", function () {
+  const inputNumber = Number(input[0]);
+  for (let i = 1; i <= +input[0]; i += 1) console.log("*".repeat(i));
```

Comment:
```suggestion
for (let i = 1; i <= +input[0]; i += 1) {
  console.log("*".repeat(i));
}
```
for문은 한줄 작성 안됩니당~~! 자바스크립트 문법에 안맞는다고 합니다.
if문은 가능 for문은 불가능!